pkg/validation/internal/testdata/valid_bundle_with_only_v1beta1_webhooks/memcached-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "cache.example.com/v1alpha1",
+          "kind": "Memcached",
+          "metadata": {
+            "name": "memcached-sample"
+          },
+          "spec": {
+            "size": 1
+          }
+        }
+      ]
+    capabilities: Basic Install
+  name: memcached-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Memcached is the Schema for the memcacheds API
+      displayName: Memcached
+      kind: Memcached
+      name: memcacheds.cache.example.com
+      version: v1alpha1
+  description: Memcached Operator description. TODO.
+  displayName: Memcached Operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      clusterPermissions:
```

Comment:
Nit: to reduce the amount of testdata, remove extraneous config like permissions/deployments (or at least use minimal values).

Reply:
The test data scenarios here are re-used for more than one check. So, I think is better we leave that since it might be useful for another case.

Reply:
If those tests need permissions etc., then they can be added later. Right now they don't. However its just a nit so feel free to ignore.